posts/2020/11/Design In Open Source

```diff
@@ -0,0 +1,39 @@
+In my experience, it’s easier to design with a specific, prioritized, goal and user base in mind, than to tackle a giant web of problems that have varying degrees of dependency on one another. And without some consensus around what will be prioritized next, or who will follow through to implement things like a refreshed User Interface, it’s risky to commit enough for larger scale design projects. Which brings us to another interesting part of design.
+“Design is not what it looks like and feels like, it’s how it works.” - Steve Jobs
+There’s multiple elements of a great user experience, but one of them is a consistent visual design. In projects who don’t already have prolific design integration, it can be a full-time job just to reconcile the existing experience with a unified UI design system. And while this does improve the experience, it only scratches the surface of what design is able to add to a project.
+## To a brighter future
+  
+While most of my open-source experience is with Jupyter, I’d have to say that design input is desired, and well received. Different projects, depending on size and scope, have different understandings of, and interest in, design. One thing that will have to happen if designers are going to make a larger impact on open-source projects, is projects need to understand the value of design beyond simply ‘making things look good.’ Many open-source initiatives would benefit from a visual overhaul, especially of the type that leans into consistent looks for similar interactions throughout the software. But that’s just scratching the surface of the problems we’ve worked through so far, and what we’re capable of in the future. 
+At the beginning we mentioned imagining a better world as a part of being a designer, and imagining a healthier open source experience is no exception! Changing the collaboration process is a good first step.
+As a Human-Computer Interaction practitioner, I always strive to understand ‘why’ a problem exists before I decide what to do about it. There’s a lot of ‘what’ and ‘how’ in open-source development. Oftentimes a scientist or developer simply solves the problem they have. But if we take some time to walk through the design process and understand more about why the question arose and who else has the problem, we can get to the core of the problem, and design both a solution and interface for it in a way that is knowledgeable both of the problem and the people solving it. In essence we build software that the user can partner with to solve their problems, instead of building a tool that accomplishes a single task.
```

Comment:
Add initials.
```suggestion
**TG:** As a Human-Computer Interaction practitioner, I always strive to understand ‘why’ a problem exists before I decide what to do about it. There’s a lot of ‘what’ and ‘how’ in open-source development. Oftentimes a scientist or developer simply solves the problem they have. But if we take some time to walk through the design process and understand more about why the question arose and who else has the problem, we can get to the core of the problem, and design both a solution and interface for it in a way that is knowledgeable both of the problem and the people solving it. In essence we build software that the user can partner with to solve their problems, instead of building a tool that accomplishes a single task.
```